inventory/byo/hosts.aep.example

```diff
@@ -180,6 +180,18 @@ openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true',
 # Configure dnsIP in the node config
 #openshift_dns_ip=172.30.0.1
 
+# NFS Options
```

Comment:
We need to update this to Persistent storage options and have a subsection for NFS